invenio_app_ils/eitems/api.py

```diff
@@ -57,6 +57,12 @@ def validate(self, record, **kwargs):
 class EItem(IlsRecord):
     """EItem record class."""
 
+    EITEM_TYPES = [
```

Comment:
not a strong opinion, I would like to also check with the others (could you discuss on a standup?): would it be more flexible to store these types in a vocabulary?